opm/parser/share/keywords/000_Eclipse100/C/COMPDAT

```diff
@@ -6,7 +6,7 @@
     {"name" : "K2" , "value_type" : "INT" },
     {"name" : "STATE" , "value_type" : "STRING" , "default" : "OPEN"},
     {"name" : "SAT_TABLE" , "value_type" : "INT" , "default" : 0},
-    {"name" : "CONNECTION_TRANSMISSIBILITY_FACTOR"     , "value_type" : "DOUBLE", "dimension" : "Viscosity*LiquidVolume/Time*Pressure"},
+    {"name" : "CONNECTION_TRANSMISSIBILITY_FACTOR"     , "value_type" : "DOUBLE", "dimension" : "Viscosity*ReservoirVolume/Time*Pressure"},
```

Comment:
that's a question best answered by joakim, but yes there is an interpreter for dimension strings. (it has a few constraints like no support for brackets and only a single division, IIRC.) basically everrything which comes before the `/` is evaluated indepenently from the stuff right to it and then the two results are divided. I could be wrong, though...